### PR TITLE
Added normal submit button on the search form

### PR DESCRIPTION
**Pull request by eugenebiz**
Moved the bizybox css to the css/plugin-global.css
Added extra checks on API fail so there will not be any PHP errors popping up

### CODE REVIEW

File: Bizyhood/Views/listings/index.php [hunk 1]
```diff
@@ -12,6 +12,7 @@
         <input type="hidden" name="cf" value="<?php echo (urldecode($_GET['cf'])); ?>">
         <?php } ?>
         <input type="search" class="bh_search-field" placeholder="Search businesses..." name="keywords" title="Search for:" value="<?php echo $keywords; ?>">
+        <input type="submit" value="&#x1f50d;">
```
Comment (eugenebiz) [inline Bizyhood/Views/listings/index.php]: This should be <button type="submit">.... for maximum flexibility in layout and decoration.

File: Bizyhood/Views/listings/promotions.php [hunk 2]
```diff
@@ -6,6 +6,7 @@
     <form action="<?php echo site_url(); ?>/index.php" method="get">
         <input type="hidden" name="page_id" value="<?php echo $list_page_id; ?>">
         <input type="search" class="bh_search-field" placeholder="Search businesses..." name="keywords" title="Search for:" value="">
+        <input type="submit" value="&#x1f50d;">
```
Comment (eugenebiz) [inline Bizyhood/Views/listings/promotions.php]: Ditto.

File: Bizyhood/Public/css/plugin.css [hunk 3]
```diff
@@ -116,7 +109,28 @@ a.facet_helper {
 
 .bh_business-search input[type="search"] {width:100%;max-width: 100%;line-height:2;padding:.5em;-webkit-box-sizing: border-box;-moz-box-sizing: border-box;box-sizing: border-box;}
 .bh_business-search {position:relative;float: right;}
-.bh_business-search:before {font-family: 'entypo';content: "\1f50d";position:absolute;right:1.25em;top:.5em;opacity:.5;font-size:20px;font-style: normal;font-weight: 400;line-height: 1;-webkit-font-smoothing: antialiased;}
+.bh_business-search input[type="submit"] {
```
Comment (eugenebiz) [inline Bizyhood/Public/css/plugin.css]: I think you should use the 'btn' class from bootstrap, but not sure. Definitely not the input selector.

File: Bizyhood/Public/css/plugin.css [hunk 4]
```diff
@@ -116,7 +109,28 @@ a.facet_helper {
...
+  font-weight: 400;
+  line-height: 1;
+  -webkit-font-smoothing: antialiased;
+  color: #000;
+  background-color: rgba(0, 0, 0, 0);
+  border: 0;
+  box-shadow: none;
+  padding: 5px; 
+}
+.bh_business-search input[type="submit"]:hover {
```
Comment (eugenebiz) [inline Bizyhood/Public/css/plugin.css]: I think you should use the 'btn' class from bootstrap, but not sure. Definitely not the input selector.

File: Bizyhood/Core.php [hunk 5]
```diff
@@ -1099,12 +1099,12 @@ public function businesses_information($atts)
       }  
       
       // avoid throwing an error
-      if (!is_array($response) || empty($response)) { return; }
+      if (!is_array($response) || (is_array($response) && isset($response['code']) && $response['code'] != 200)) { return; }
```
Comment (eugenebiz) [inline Bizyhood/Core.php]: Looks good!

File: Bizyhood/Views/listings/index.php (hunk 1 above)
Comment (spyrosvl) [inline Bizyhood/Views/listings/index.php]: Can you please explain in more detail?

File: Bizyhood/Public/css/plugin.css (hunk 3 above)
Comment (spyrosvl) [inline Bizyhood/Public/css/plugin.css]: I will add a separate class. btn is not good here. It will add styling that I will need to neutralize.

**Pull request merged**